.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,50 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: actions/checkout@v3
+      # Ginkgo and gomega are not required to pass linter.
```

Comment:
If they aren't needed then my preference would be to omit them until they are needed. The action will run faster, be easier to understand, and be less likely to hit dependency issues.

Reply:
True. I added them only to have some additional test for dependencies in this project. They maybe still useful, but belong to a different action.